.github/workflows/test-integrations.yml

```diff
@@ -265,7 +265,7 @@ jobs:
     strategy:
       fail-fast: false
       matrix:
-        envoy-version: ["1.22.11", "1.23.8", "1.24.6", "1.25.4"]
+        envoy-version: ["1.21.6", "1.22.11", "1.23.9", "1.24.7"]
```

Comment:
I think the inclusion of 1.25 instead of 1.21 in this file was just a consequence of backports when the original was added. This seems correct to me to bring CI back in line w/ the rest of the code and docs.